brooklinevoiceapp/mycity/test/integration_tests/test_fallback_intent.py

```diff
@@ -0,0 +1,19 @@
+"""Integration tests for fallback intent"""
+
+from mycity.intents import fallback_intent
+from mycity.test import test_constants
+from mycity.test.integration_tests import (
+    intent_base_case as base_case,
+    intent_test_mixins as mix_ins,
+)
+
+class TrashPickupTestCase(mix_ins.RepromptTextTestMixIn,
```

Comment:
Rename to `FallbackIntentTestCase`

Reply:
I now changed the fallback intent test case to the given name.